lib/jobs/dell-wsman-import-scp.js

```diff
@@ -123,13 +123,19 @@ function DellWsmanImportSCPJobFactory(
                 logger.info('Response from SCP Microservice for Import System Configuration: ' +  response.body);
                 var json = JSON.parse(response.body);
                 logger.info('Status from SCP Microservice for Import System Configuration:  ' + json["status"]);
-                if(json["status"]=="OK") {
-                    return response;
+
+                if (json["status"]==="OK" && json.hasOwnProperty("xmlConfig") && null !== json["xmlConfig"]) {
```

Comment:
I see this code block is duplicated across multiple files, suggest creating a function for it.